### PR TITLE
patch(PDC fetching) Except warnings from the dsmc command

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
The `cg.utils.commands.Process.run_command` raises an error on all exit codes except 0 (success). This causes problems in the PDC flow since the dsmc command regularly gives warnings, which result in exit code 8. These are non-fatal and should not abort the command.

This PR catches `CalledProcessError` exceptions checks the return code and does not raise an error if it is 8.

### Changed

- Warning exit codes from the dsmc command no longer raise errors

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b fc-fetch-except-warnings
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/backup/pdc.py [hunk 1]
```diff
@@ -76,8 +76,11 @@ def run_dsmc_command(self, command: list) -> None:
         LOG.debug(f"{self.process.binary} {' '.join(command)}")
         try:
             self.process.run_command(parameters=command, dry_run=self.dry_run)
-        except Exception as error:
-            raise PdcError(f"{error}") from error
+        except CalledProcessError as error:
+            if error.returncode == EXIT_WARNING:
```
Comment (seallard) [inline cg/meta/backup/pdc.py:80]: Nice, this ensures exit warnings do not cause the flow to abort 👍

**Review by seallard — APPROVED** [review]
Looks much cleaner and should handle the exit warnings 👍

**Comment by Vince-janv** [comment]
Retrieval and backup work on stage

**Comment by Vince-janv** [comment]
Deployed in production

**Pull request merged**